src/vma/proto/mem_buf_desc.h

```diff
@@ -130,11 +125,9 @@ class mem_buf_desc_t {
 	inline int inc_ref_count() {return atomic_fetch_and_inc(&n_ref_count);}
 	inline int dec_ref_count() {return atomic_fetch_and_dec(&n_ref_count);}
 
-#ifdef DEFINED_VMAPOLL
```

Comment:
Please restore the `#ifdef DEFINED_VMAPOLL` and remove in later commit if necessary.